client/my-sites/site-settings/section-import.jsx

```diff
@@ -138,24 +138,23 @@ class SiteSettingsImport extends Component {
 	 * @param {Array} importsForSite The list of active import jobs
 	 * @returns {Array} Importer react elements for the active import jobs
 	 */
-	renderActiveImporters( site, siteTitle, importsForSite ) {
+	renderActiveImporters( importsForSite ) {
```

Comment:
Looks like the jsdocs just need to be updated to reflect the change of params.

Reply:
Oops! Good job you and CircleCI are around. Thanks!